specification/deviceregistry/DeviceRegistry.Management/properties/ns_assetProperties.tsp

```diff
@@ -181,13 +188,43 @@ model NamespaceDatasetDataPoint is NamespaceDataPoint {
 @added(Versions.v2025_07_01_preview)
 model NamespaceEventDataPoint is NamespaceDataPoint;
 
+@doc("Defines the event group properties.")
+@added(Versions.v2025_10_01)
+model NamespaceEventGroup {
+  @doc("The name of the event group.")
+  name: string;
+
+  @doc("The address of the notifier of the event group in the asset (e.g. URL) so that a client can access the event group on the asset.")
+  dataSource?: string;
+
+  @doc("Stringified JSON that contains connector-specific configuration for the event group. For OPC UA, this could include configuration like, publishingInterval, samplingInterval, and queueSize.")
+  eventGroupConfiguration?: string;
+
+  @doc("Destinations for events. Default destinations when destinations is not defined at the event level.")
+  @identifiers(#[])
+  defaultDestinations?: EventDestination[];
+
+  @doc("URI or type definition ID.")
+  typeRef?: string;
+
+  @doc("Array of events that are part of the event group.")
+  @identifiers(#[])
+  events?: NamespaceEvent[];
+}
+
 @doc("Defines the event properties.")
 @added(Versions.v2025_07_01_preview)
 model NamespaceEvent {
   @doc("The name of the event.")
   name: string;
 
+  @doc("Reference to a data source for a given event.")
+  @added(Versions.v2025_10_01)
+  dataSource?: string;
+
   @doc("The address of the notifier of the event in the asset (e.g. URL) so that a client can access the event on the asset.")
```

Comment:
Nit - if we can merge / use the eventNotifier comment with/as the datasource comment, that would be great.